R/data_meredith.R

```diff
@@ -0,0 +1,353 @@
+#' American Health Care Act Polls
+#'
+#' The raw data behind the story
+#' "Why The GOP Is So Hell-Bent On Passing An Unpopular Health Care Bill"
+#' \url{https://fivethirtyeight.com/features/why-the-gop-is-so-hell-bent-on-passing-an-unpopular-health-care-bill}.
+#'
+#' @format A data frame with 15 rows representing polls and 7 variables:
+#' \describe{
+#'   \item{start}{Start date of the poll.}
+#'   \item{end}{End date of the poll.}
+#'   \item{pollster}{The entity that conducts and collects information from the poll.}
+#'   \item{favor}{The number of affirmative responses to the question at the pollster.}
+#'   \item{oppose}{The number of negative reponses to the question at the pollster.}
+#'   \item{url}{The website associated with the polling question.}
+#'   \item{text}{The polling question asked at the pollster.}
+#' }
+#' @source See \url{https://github.com/fivethirtyeight/data/blob/master/ahca-polls/README.md}
+#'  @examples
```

Comment:
After you've built the package, don't forget to look at the individual help files Ex `?ahca_polls`. For example, the extra space in front of `@examples` is causing the output to break

Reply:
You can click the "Build and Reload" button here to build the package. Don't forget to turn on roxygen help file building as noted [here](http://r-pkgs.had.co.nz/man.html#man-workflow-2) first

<img width="625" alt="build" src="https://user-images.githubusercontent.com/4574203/35124682-0015364a-fc75-11e7-942d-514fec6ba84b.png">